core/frontend/src/test/IModelApp.test.ts

```diff
@@ -44,36 +44,37 @@ class TestRotateTool extends RotateViewTool { }
 class TestSelectTool extends SelectionTool { }
 
 class TestApp extends MockRender.App {
-  public static testNamespace?: I18NNamespace;
+  public static testNamespace?: Promise<void>;

```

Comment:
Is this still necessary and can we just `await` the promise for registerNamespace below?